src/steps/generic.rs

```diff
@@ -916,9 +916,17 @@ pub fn run_chezmoi_update(ctx: &ExecutionContext) -> Result<()> {
     let chezmoi = require("chezmoi")?;
     HOME_DIR.join(".local/share/chezmoi").require()?;
 
+    let mut executor = ctx.execute(chezmoi);
```

Comment:
Nit, `executor` is not the best name for it
```suggestion
    let mut cmd = ctx.execute(chezmoi);
```